docs/blog/posts/cel-and-kubescape/cel-and-kubescape

```diff
@@ -0,0 +1,207 @@
+If you are reading this and thinking: “I already have OPA Gatekeeper or Kyverno up and running to solve this”.
+<figure markdown>
+  ![Admission control meme](/blog/cel-and-kubescape/admission-control-meme.jpg){ width="600" }
+ </figure>
+Please join us to get a better understanding of this Kubernetes native feature, what it seeks to achieve and what the benefits of using it are. Also, take into account that this functionality will become the de-facto standard, similar to what happened with OpenTelemetry and GatewayAPI.
+
+### Introduction to ValidatingAdmissionPolicies
+ValidatingAdmissionPolicies are: A declarative, in-process alternative to validating admission webhooks. They use the Common Expression Language (CEL) to define the policies.
+
+They consist of two main resources:
+* ValidatingAdmissionPolicy describes the policy logic
+* ValidatingAdmissionPolicyBinding links the above policy to the resources it applies to
+
+### The Goals of Validating Admission Policies
+The Kubernetes team has established specific goals for Validating Admission Policies to guarantee that it aligns with the community's requirements.:
+1. **Alternative to Webhooks**: Validating Admission Policies aims to provide an alternative to webhooks for the majority of validating admission use cases.
+2. **In-Tree Extensions**: The feature intends to offer the in-tree extensions required to build policy frameworks for Kubernetes without relying on webhooks.
+3. **Efficient Type Checking**: Validating Admission Policies aims to make good use of CEL type checking, even when considering that CRD schemas can be changed at any time, and not all fields of built-in types exist in older Kubernetes versions.
+4. **Polyfill Implementati****on**: The Kubernetes organization plans to provide a polyfill implementation that offers enhancement functionality to Kubernetes versions where this feature is not available.
+5. **Core Functionality as a Library**: Validating Admission Policies intends to provide core functionality as a library, enabling use cases like GitOps, CI/CD pipelines, and auditing to run the same CEL validation checks as the API server.
+Having said that, Polyfill Implementation and Core Functionality as a Library (points 4 and 5 above) have yet to be implemented, and we should keep an eye out for them in the future.
+
+### The Benefits of Validating Admission Policies
+The Validating Admission Policies functionality offers several advantages over webhooks, making it a more attractive choice for policy enforcement:
+1. **No Infrastructure Requirement**: Validating Admission Policies doesn't require you to build infrastructure to host the admission webhook, simplifying the process.
+2. **Reduced Latency**: By eliminating the need for an additional network hop, Validating Admission Policies helps reduce latency, ensuring smoother and faster operations.
```

Comment:
I would add here scalability. Kubernetes control plane scalability does not require the scalability of a non-control plane component (the webhook implementation).

Reply:
You mean add explicitly?
Becasue this can be inferred from the first bullet.